src/Resources/assets/styles/_tailwind.scss

```diff
@@ -41,6 +41,10 @@
         @apply border-none;
     }
 
+    .whatwedo_table-wrapper {
```

Comment:
Nit: I wonder if we should apply these default styles with a `:where` selector, so they are naturally made to be overwritten easier. The `:where` applies a lower specificity by default.

Reply:
I noted this. Good input. Can be discussed later since it's not in scope for this ticket.